Main.elm

```diff
@@ -50,18 +50,16 @@ distractionTypes =
     ]
 
 
-init : Maybe Model -> ( Model, Cmd a )
-init flags =
-    case flags of
-        Just model ->
-            ( model, Cmd.none )
-
-        Nothing ->
-            ( { distractions = []
-              , version = "1"
-              }
-            , Cmd.none
-            )
+emptyModel : Model
+emptyModel =
+    { distractions = []
+    , version = "1"
+    }
+
+
+init : Maybe Model -> ( Model, Cmd Msg )
+init savedModel =
+    Maybe.withDefault emptyModel savedModel ! []
```

Comment:
This is awesome!  I went looking for Maybe.withDefault and somehow didn't find it...  I double checked it did exist on maybe in core 4.0.5, so I just missed it some how.

Also, I hadn't see the (!) operator, that's convenient!

Thanks!